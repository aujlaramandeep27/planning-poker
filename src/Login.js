import React from 'react';
import logo from './images/logo.png';
import {Link} from 'react-router-dom';

export default class Login extends React.Component {
    
    render() {
        return (
            <div className="uk-flex uk-flex-column uk-flex-middle">
                <img src={logo} alt="logo" height="40vmin" uk-img className="uk-animation-kenburns"/>
                <input placeholder="EMAIL" className="uk-margin-top"></input>
                <input placeholder="PASSWORD" className="uk-margin"></input>
                <Link to="/rooms/" className="uk-button uk-button-secondary">LOGIN</Link>
            </div>
        );
    }    
}