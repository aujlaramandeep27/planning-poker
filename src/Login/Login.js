import React from 'react';
import logo from '../images/logo.png';
import {Link} from 'react-router-dom';

export default function Login() {
    
    return (
        <div className="uk-flex uk-flex-center">
            <div className="uk-card uk-card-default uk-card-body uk-width-1-2@m">
                <img src={logo} alt="logo" height="40vmin" data-uk-img className="uk-animation-kenburns uk-align-center"/>
                <h1>Planning Poker</h1>
                <input placeholder="EMAIL" className="uk-input uk-margin-top" type="email"></input>
                <input placeholder="PASSWORD" className="uk-input uk-margin" type="password"></input>
                <Link to="/rooms/" className="uk-button uk-button-primary">LOGIN</Link>
            </div>
        </div>
    );
}