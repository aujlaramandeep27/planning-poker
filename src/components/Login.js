import React from 'react';
import { Link } from 'react-router-dom';

export default function Login() {
  return (
    <>
      <h1>Planning Poker</h1>
      <form>
        <div className="uk-margin">
          <div className="uk-inline uk-width-1-1">
            <span className="uk-form-icon" data-uk-icon="icon: mail"/>
            <input className="uk-input" type="text"/>
          </div>
        </div>

        <div className="uk-margin">
          <div className="uk-inline uk-width-1-1">
            <span className="uk-form-icon" data-uk-icon="icon: lock"/>
            <input className="uk-input" type="password"/>
          </div>
        </div>

        <div className="uk-margin">
          <Link to="/rooms/" className="uk-button uk-button-primary uk-width-1-1">Login</Link>
        </div>
      </form>
    </>
  );
}
