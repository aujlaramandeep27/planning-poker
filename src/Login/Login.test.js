import Login from "./Login";

it('renders correctly', () => {
    expect(new Login()).toMatchSnapshot(); 
});