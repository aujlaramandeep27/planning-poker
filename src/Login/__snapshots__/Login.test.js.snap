// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Login {
  "context": undefined,
  "props": undefined,
  "refs": Object {},
  "updater": Object {
    "enqueueForceUpdate": [Function],
    "enqueueReplaceState": [Function],
    "enqueueSetState": [Function],
    "isMounted": [Function],
  },
}
`;
